packages/utils/src/index.ts

```diff
@@ -1,5 +1,6 @@
 export { AccountUtils } from './account'
 export { AttestationUtils } from './attestations'
+export * from './celoHistory'
```

Comment:
It's preferable to not export * from files to avoid polluting the namespace. I realize we already do this below but something we should avoid doing again going forward.